src/main/java/se/myhappyplants/client/view/SearchPlantPane.java

```diff
@@ -79,5 +94,34 @@ public void setDefaultImage(String defaultImage) {
         Image img = new Image(defaultImage);
         image.setImage(img);
     }
+
+    public void extendPaneMoreInfoPlant() {
+
+        Timeline timeline = new Timeline(
+                new KeyFrame(Duration.millis(100), event -> this.setPrefHeight(this.getHeight() + 50))
+        );
+        timeline.setCycleCount(4);
+        timeline.play();
+        timeline.setOnFinished(action -> this.getChildren().addAll());
```

Comment:
Ett tips för att undvika buggar är att -istället för att varje gång användaren klickar expand lägga till alla knappar, sen ta bort dem igen, lägga till dem, ta bort dem osv -lägga till raden this.getChildren().addAll(); en enda gång precis ovanför metoden. För annars riskar man att få exception tillslut och att knapparna försvinner plötsligt :)